ecs-cli/modules/utils/compose/ecs_params_reader_test.go

```diff
@@ -145,6 +151,9 @@ task_definition:
 
 		assert.ElementsMatch(t, expectedSecrets, wordpress.Secrets, "Expected secrets to match")
 		assert.ElementsMatch(t, expectedSecrets, wordpress.Logging.SecretOptions, "Expected secrets to match")
+
+		assert.Equal(t, "fluentbit", log_router.FirelensConfiguration.Type, "Exception firelens_configuration to be fluentbit")
+		assert.Equal(t, "true", log_router.FirelensConfiguration.Options["enable-ecs-log-metadata"], "Exception firelens_configuration to be fluentbit")
```

Comment:
"Exception" => "Expected"

And the second one should say something like "Expected 'enable-ecs-log-metadata" to be 'true'"